workshop/Lab2/README.md

```diff
@@ -0,0 +1,144 @@
+# Lab 2 Deploy an application with a manually provisioned backing store
+
+Learn to manually provision a Redis instance, and manually connect it to an
+application hosted on the IBM Kubernetes Service.
+
+# Deploy your application
+
+If you're continuing from Lab 1, you can skip this step. Otherwise, run
+
+```
+$ kubectl run guestbook --image=ibmcom/guestbook:v1
```

Comment:
Not made it all the way thru yet but do we ever actually use the yaml from the guestbook repo? If not, why not? And, if not, why did we clone the repo?

Reply:
@jberkhahn ^^

Reply:
I fixed this didnt I? I removed the instruction to download because we never actually used it for anything.